src/plugins/intel_cpu/tests/functional/subgraph_tests/src/ngram.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Also, I would recommend to you to add one more test that will prevent the common transformations to break the pattern that we expect for Ngram now.
You can add TF FE unit-test with extracted sub-graph of `Ngram` in pbtxt and pass it into core read and compile and see if your pattern will apply. Because now we have some risk that some previous transformation will break it.

Reply:
The test is added, thanks for the suggestion. Could you please take a look?